### DISCUSSION TITLE
Regarding getVerboseDateTimeRepresentation function

### DISCUSSION THREAD

**Discussion by itaishalom** [flyerhq/flutter_chat_ui#120] [Q&A] [answered]
Hi there, so let's say I wish that the date divider will look more like whatsassp - in the sense of showing it once a day. Is it possible?
Right now this is the problem:

` nextMessageDateThreshold =
          nextMessage!.createdAt! - message.createdAt! >= 900000;`

Is it possible to change it in to comparing dates?
Maybe make it a custom function?
I can do it, should I PR?

**Comment by demchenkoalex** [comment]
No, but it can be quickly added, today or tomorrow for example.

It will not be a custom function, but threshold will be configurable.

**Reply by itaishalom** [reply]
Thanks again for your reply. 
I'll PR for a fix, something custom.

**Reply by demchenkoalex** [reply]
No need, I will do it myself, I need to verify a couple of things :)

**Reply by itaishalom** [reply]
Ok cool thank you! because 15 minutes is redundant for our usage.
Thank you!

**Comment by demchenkoalex** [comment, marked as answer]
Released in [v1.4.4](https://github.com/flyerhq/flutter_chat_ui/releases/tag/v1.4.4). Just set the `dateHeaderThreshold` to something big (for example 86,400,000 - ms in a day), and date header will only appear if time between 2 messages is less than this value. Not related to new dates, date header will always appear on a new date.

**Reply by itaishalom** [reply]
Hi, thank you for your time.
Before driving into the code, quick question: if one message is sent on 23:59 and the second sent one minute later, will i still see the date?

**Reply by demchenkoalex** [reply]
New day = date header, that's pretty much every messenger that I used, so yes, and it will not be customizable :)

**Reply by itaishalom** [reply]
Dude, you did an incredible code, just read. Awesome work.
Thank you.
